docs/resources/aws_guardduty_detectors.md

```diff
@@ -30,11 +30,11 @@ See also the [AWS documentation on GuardDuty Detectors](https://docs.aws.amazon.
 
 |Property                 | Description|
 | ---                     | --- |
-|detector_ids             | The architecture of the image. |
+|detector_ids             | A list of detector IDs. |
 
 ## Examples
 
-##### Ensure ID of an AMI GuardDuty Detector
+##### Ensure ID of a GuardDuty Detector
```

Comment:
Perhaps something along these lines would read better...
`Ensure GuardDuty Detector ID exists` 
OR
`Ensure GuardDuty Detector ID is returned`

Reply:
updated